vultr/resource_vultr_vpc2.go

```diff
@@ -0,0 +1,148 @@
+package vultr
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"strings"
+	"time"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/retry"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
+	"github.com/vultr/govultr/v3"
+)
+
+func resourceVultrVPC2() *schema.Resource {
+	return &schema.Resource{
+		CreateContext: resourceVultrVPC2Create,
+		ReadContext:   resourceVultrVPC2Read,
+		UpdateContext: resourceVultrVPC2Update,
+		DeleteContext: resourceVultrVPC2Delete,
+		Importer: &schema.ResourceImporter{
+			StateContext: schema.ImportStatePassthroughContext,
+		},
+
+		Schema: map[string]*schema.Schema{
```

Comment:
We need to include `ip_type` to the schema as well.  https://www.vultr.com/api/#operation/create-vpc2

Reply:
Sorry. Fixed.